pkg/server/svid/rotator_config_test.go

```diff
@@ -2,10 +2,14 @@ package svid
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestNewRotator(t *testing.T) {
-	r := newRotator(&RotatorConfig{})
+	rotatorInterface := NewRotator(&RotatorConfig{})
+	r, ok := rotatorInterface.(*rotator)
```

Comment:
Hmm, i wonder if it's time to remove the `Rotator` interface and rename `rotator` to `Rotator`? The interface isn't useful. As far as observing that a default interval is being used, we could add an `Interval() time.Duration` method to `*Rotator` that the unit tests could use to assert that the default is properly applied.

Reply:
Good idea!